.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     needs: ['rubocop']
     strategy:
       matrix:
-        ruby-version: ['3.0', '3.1', '3.2', '3.3', 'head']
+        ruby-version: ['3.0', '3.1', '3.2', '3.3', '3.4']
```

Comment:
remove 3.0, as you mentioned in the description, and maybe replacing head by 3.5-dev

Reply:
We currently have 3.0 as a requirement in the gemspec.
What if I just change the PR description for now. And add 3.5-dev with ruby-head in a separate pull request where we can focus on supporting exactly 3.5?